pennylane_lightning/core/src/simulators/lightning_gpu/gates/tests/Test_StateVectorCudaManaged_Generators.cpp

```diff
@@ -1662,3 +1662,324 @@ TEMPLATE_TEST_CASE("StateVectorCudaManaged::applyGeneratorGlobalPhase",
         }
     }
 }
+
+TEMPLATE_TEST_CASE("Generators::applyControlledGenerator - 1 wire",
+                   "[GateGenerators]", float, double) {
+
+    const std::string controlled_gate_name =
+        // GENERATE("RX", "RY", "RZ", "PhaseShift", "GlobalPhase", "MultiRZ");
+        "MultiRZ";
+    const std::size_t num_qubits = 4;
+    const TestType ep = 1e-3;
+    const TestType EP = 1e-4;
```

Comment:
I would prefer not to have two variables with similar names. Could you please rename them to be more descriptive?

Reply:
Good idea, I will change it.